test/spec.index.js

```diff
@@ -250,6 +250,57 @@ describe('Template Mixins', function () {
                 }));
             });
 
+            it('by default, assumes the field isn\'t required', function () {
```

Comment:
In that case, should form-wizard not default to applying the 'required' validator? i.e.That the user of the library must actively state that the field is optional?

It seems to me that no matter how you look at it, there is a mismatch between how the two libraries work; template-mixins defaults to required, form-wizard / form-controller defaults to optional.

Should these libraries be attempting to enforce these government standards? Is that not the job of the team building the individual form?